### PR TITLE
add vmargs=-XX:-UseContainerSupport in config

### PR DESCRIPTION
**Pull request by lxning**
*Issue #, if available:*
#99 

*Description of changes:*
Apply the [fixing](https://github.com/aws/sagemaker-inference-toolkit/pull/87) in pytorch inference toolkit

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by davidthomas426** [comment]
I'll just note that this issue along with workarounds and fixes have shown up across different inference toolkits.

Here is a list of links showing that it's been a recurring problem.

We should make sure to address this uniformly across the inference toolkits and deep-learning-containers, while allowing users to easily customize without needing onerous workarounds such as using derived deep-learning-container images or even forking toolkit.

Links:
- SageMaker docs troubleshooting:
  - https://docs.aws.amazon.com/sagemaker/latest/dg/deploy-model-troubleshoot.html
- Other issues, across inference toolkits:
  * Base Inference Toolkit: https://github.com/aws/sagemaker-inference-toolkit/issues/82
  * HuggingFace toolkit: https://github.com/aws/sagemaker-huggingface-inference-toolkit/issues/3
  * 
- Other pull request from this repo:
  * Turn off UseContainerSupport: https://github.com/aws/sagemaker-pytorch-inference-toolkit/pull/98
  * Same?: https://github.com/aws/sagemaker-pytorch-inference-toolkit/pull/100
- Other pull requests, other toolkits:
  * https://github.com/aws/sagemaker-inference-toolkit/pull/83
  * Allow user to change vmargs: https://github.com/aws/sagemaker-inference-toolkit/pull/118
    - Note previous hack, requiring derived container (but not forked toolkit), using DockerFile change:
       - https://github.com/aws/sagemaker-inference-toolkit/issues/82#issuecomment-802234692
       - https://github.com/aws/sagemaker-pytorch-inference-toolkit/issues/99#issuecomment-968695239
  * https://github.com/aws/deep-learning-containers/pull/1077
    - @vdantu mentioned this in PR code review, but it was ignored twice

- DLC repo still has vmargs with +UseContainerSupport in three different places:
  * https://github.com/aws/deep-learning-containers/blob/master/autogluon/build_artifacts/inference/config.properties
  * https://github.com/aws/deep-learning-containers/blob/master/mxnet/inference/docker/artifacts/config.properties
  * https://github.com/aws/deep-learning-containers/blob/master/pytorch/inference/docker/build_artifacts/config.properties

**Comment by chen3933** [comment]
The PR will be updated to allow customization of vmargs
Example : 
https://github.com/aws/sagemaker-inference-toolkit/pull/118/commits/c01dde749687f86e7891ec403eed3f98d4fcfb50

**Comment by rohithkrn** [comment]
python 3.7 tests failing at coverage report step. Failing to invoke coverage command. Works fine for python3.6
`ERROR: InvocationError for command /codebuild/output/src309395522/src/github.com/aws/sagemaker-pytorch-inference-toolkit/.tox/py37/bin/coverage report --fail-under=90 --include '*sagemaker_pytorch_serving_container*' (exited with code 1)`

**Review by chen3933 — DISMISSED** [review]

File: tox.ini [hunk 1]
```diff
@@ -55,7 +55,7 @@ commands =
     {env:IGNORE_COVERAGE:} coverage report --fail-under=90 --include *sagemaker_pytorch_serving_container*
 
 deps =
-    coverage
+    coverage==6.2
```
Comment (davidthomas426) [inline tox.ini:58]: Can't we instead update API to coverage 7? I think all you need to do is remove the `--include *sagemaker_pytorch_serving_container*` part from line 55, which isn't really needed anyway given the `--source` argument in line 54.

Comment (chen3933) [reply tox.ini:58]: +1

Comment (maaquib) [reply tox.ini]: @davidthomas426 Done

**Review by davidthomas426 — APPROVED** [review]

**Pull request merged**